scripts/get_tool_dl_url.py

```diff
@@ -73,7 +73,7 @@ class Nooba:
     "Nooba's URL pattern."
 
     repo = "noobaa/noobaa-operator"
-    arch = "mac" if OS == "Darwin" else "linux"
+    arch = OS.lower()
     pattern = re.compile(f"https://(.*)-{arch}-(.*)[0-9]")
```

Comment:
I think this can break for arm (noobaa url changed for versions higher than 5.11 https://github.com/noobaa/noobaa-operator/releases)

But if the change I suggested in `scripts/install_dev_tools.sh` works for you, maybe we can ignore this change, because I want to change this script in the future to be like a dependabot (looking for updates, like https://github.com/dora-metrics/pelorus/blob/master/.github/workflows/ci.yml looks for OpenShift updates)

Reply:
Can you elaborate a bit on how that script works?

Reply:
The github action looking for OpenShift version updates? It looks an url that returns a json and checks if we are using last 4 minor stable releases (for version 4) in the project (looking in some doc files for example) and also in PROW repo, to check that we are running CI also in last 4 minor stable releases